gift-certificates-app/web/src/main/java/com/epam/esm/exception/MessageLocal.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.exception;
+
+public interface MessageLocal {
+
+    String MESSAGE_SOMETHING_WRONG = "message.somethingWrong";
+    String MESSAGE_ID_MIN = "message.path.id.min";
+
+    String MESSAGE_RESOURCE_NOT_FOUND = "message.resource.notFound";
+    String MESSAGE_RESOURCE_ALREADY_EXISTS = "message.resource.alreadyExists";
+    String MESSAGE_RESOURCE_HAS_LINKS = "message.resource.hasLinks";
+    String MESSAGE_RESOURCE_NO_LINKS = "message.resource.noLinks";
+
+    String MESSAGE_TOKEN_INVALID = "message.token.invalid";
```

Comment:
Looks like MESSAGE_TOKEN_INVALID and MESSAGE_TOKEN_EXPIRED not used and could be deleted

Reply:
Oh, sorry, I missed it. Please ignore my comment then